### DISCUSSION TITLE
Thank you to the developers

### DISCUSSION THREAD

**Discussion by FyeUK** [pybricks/support#988] [General]
Hi all,

I just wanted to say thank you to the developers of Pybricks. 

I've only today started to experiment with it - using it to automate some of my Lego City trains and using Python for the first time, and I have got to say that I'm seriously impressed with how powerful and well documented it is. 

I'm an IT Consultant in the real world so I'm not new to programming or technical stuff... but I must say, I work with commercialised products that are sold B2B for not insignificant sums of money that aren't anywhere near as well put together as this.

Seriously well done to the devs, I can't wait to play with it more in future 👍

**Comment by laurensvalk** [comment]
Glad to hear you enjoy using Pybricks. And thanks for becoming a Patreon sponsor!
